.github/workflows/e2e-tests.yml

```
@@ -2,27 +2,39 @@ name: e2e-tests
 on:
   push:
     branches: [main]
-  pull_request:
+  pull_request_target:
```

Restrict to main branch:

```suggestion
  pull_request_target:
    branches: main
```